pyodide-build/pyodide_build/buildpkg.py

```diff
@@ -81,7 +84,7 @@ def run(self, cmd, **opts):
 
     def close(self):
         """Free the file descriptors."""
-        if self._fd_read:
+        if self._fd_read and self._fd_write:
```

Comment:
Both should always be true, and this protects the later part for mypy. This would be valid place for an assert, instead.